### PR TITLE
Add example using the module in a test

### PR DESCRIPTION
**Pull request by paulcsmith**

### CODE REVIEW

File: lib/ex_machina.ex [hunk 1]
```diff
@@ -4,6 +4,8 @@ defmodule ExMachina do
 
   ## Examples
 
+  Put this somewhere like `test/support/ex_machina.ex`
```
Comment (BlakeWilliams) [inline lib/ex_machina.ex]: `Create a file in your application for ex machina: e.g. `test/support/ex_machina.ex``?

Comment (paulcsmith) [inline lib/ex_machina.ex]: Not sure about that. It is more specific, but longer too. What did you dislike about "Put this somewhere like `test/support/ex_machina.ex`?

Comment (paulcsmith) [inline lib/ex_machina.ex]: Maybe `Create a module for your factories. Often somewhere like `test/support/machina.ex`?

Comment (BlakeWilliams) [inline lib/ex_machina.ex]: I like, that, thoughts on `e.g.` instead of `Often somewhere like`?

**Comment by BlakeWilliams** [comment]
LGTM!

**Pull request merged**